packages/ncids-css/packages/ncids/_index.scss

```diff
@@ -4,39 +4,56 @@
 // -------------------------------------
 @forward 'uswds-global';
 
-// Helpers
-// -------------------------------------
-@forward 'uswds-helpers';
-
-// Components
+// Typography & Form Controls
 // -------------------------------------
 @forward 'uswds-typography';
 @forward 'uswds-form-controls';
 
+// NCIDS
+// -------------------------------------
 @forward 'nci-autocomplete';
+@forward 'nci-back-to-top';
```

Comment:
Do we need this if we already have the footer?

Reply:
gone